messaging/messaging-core/src/test/java/uk/gov/justice/services/messaging/JsonObjectsTest.java

```diff
@@ -299,4 +303,58 @@ public void shouldCreateBuilderFromJsonObjectWithFilter() {
         assertThat(actual.getString("name"), equalTo(source.getString("name")));
     }
 
+    @Test
+    public void shouldConvertCollectionOfJsonObjectsToJsonArray() {
+
+        final String oldKey = "oldKey";
+        final String newKey = "newKey";
+
+        JsonArray input = Json.createArrayBuilder()
```

Comment:
Why do we create a JsonArray here to use as the input, when it then gets converted to a normal List before use? Couldn't we just create a List in the first place? I think this would make the test less confusing.